clients/src/main/java/org/apache/kafka/clients/ClientRequest.java

```diff
@@ -85,11 +89,12 @@ public ApiKeys apiKey() {
     public RequestHeader makeHeader(short version) {
         short requestApiKey = requestBuilder.apiKey().id;
         return new RequestHeader(
-            new RequestHeaderData().
-                setRequestApiKey(requestApiKey).
-                setRequestApiVersion(version).
-                setClientId(clientId).
-                setCorrelationId(correlationId),
+            new RequestHeaderData()
```

Comment:
it would be good to use the existing style of periods at the end to avoid a big change here